ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -337,15 +389,21 @@ SQUtils.QObject {
     UnblockWithSeedphraseFlow {
         id: unblockWithSeedphraseFlow
 
+        property string pin
```

Comment:
What's this for? 🤔 And why not inside of `UnblockWithSeedphraseFlow`?

Reply:
It's there because it's a details of the `OnboardingFlow`, not the `UnblockWithSeedphraseFlow`. It's intercepted because  it's required by `loginRequested` signal.